cypress/e2e/minet.cy.ts

```diff
@@ -0,0 +1,79 @@
+import { Dependencies } from "../robots/Minet/Basic.cy";
+import { DashboardEyes, DashboardHands } from "../robots/Minet/Dashboard.cy";
+import { LoginEyes, LoginHands } from "../robots/Minet/Login.cy";
+import {
+  SignUpPageRobotEyes,
+  SignUppageRobotHands,
+} from "../robots/Minet/Signup.cy";
+import { TradeEyes, TradeHands } from "../robots/Minet/Trade.cy";
+
+const dependancies = new Dependencies();
+const signupPageEyes = new SignUpPageRobotEyes();
+const signUppageHands = new SignUppageRobotHands();
+const loginPageEyes = new LoginEyes();
+const loginPageHands = new LoginHands();
+const dashboardPageEyes = new DashboardEyes();
+const dashboardPageHands = new DashboardHands();
+const tradePageEyes = new TradeEyes();
+const tradePageHands = new TradeHands();
+
+context("Test application", () => {
+  describe("Minet E2E Testing", () => {
+    it("starts the testing", () => {
+      dependancies.viewPort();
+      dependancies.accessUrl(Cypress.env("host_url"));
+
+      signupPageEyes.AssertTitle();
+      signUppageHands.EnterName();
+      signUppageHands.EnterEmail();
+      signUppageHands.EnterPassword();
+      signupPageEyes.AssertCaption();
+      signUppageHands.ClickSignUp();
+      dependancies.wait();
+      dashboardPageHands.Logout();
+
+      dependancies.accessUrl(Cypress.env("host_url") + "login");
```

Comment:
remove hard coding of env urls in robot file, values should pass from spec files

Reply:
done